### ISSUE TITLE
Anti-thyroid-stimulating hormone receptor antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7355]
**Preferred term label:**
Anti-thyroid-stimulating hormone receptor antibody positivity

**Synonyms**
Anti-TSH receptor antibody positivity, anti-thyroid-stimulating hormone receptor antibody positivity, TRAbs,

**Definition (free text, please give PubMed ID)**
PMID: 10874533, PMID: 30224035, PMID: 3018097, PMID: 26293122

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
autoimmune thyroid diseases and Graves' disease. Graves-Basedow disease

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by MaudDLVT** [comment]
@julienN75

**Comment by pnrobinson** [comment]
added as HP:0034189
